keps/sig-node/753-sidecar-containers/README.md

```diff
@@ -1272,13 +1544,20 @@ and operation of this feature.
 Recall that end users cannot usually observe component logs or access metrics.
 -->
 
-- [ ] Events
-  - Event Reason: 
-- [ ] API .status
-  - Condition name: 
-  - Other field: 
-- [ ] Other (treat as last resort)
-  - Details:
+- [X] Other (treat as last resort)
+  - Details: `kubectl describe pod <pod-name>` will show the new field `.spec.initContainers[i].restartPolicy` for the sidecar containers.
```

Comment:
That doesn't mean the feature is working - that means that I wanted it to work.

What we probably want to check is if the init container with restartPolicy set is running together with the regular containers.